util/Migrator/DbScripts/2022-03-01_00_AddApiKeysTable.sql

```diff
@@ -488,28 +488,6 @@ FROM
     [dbo].[Organization]
 GO
 
-
-IF OBJECT_ID('[dbo].[OrganizationSponsorship_ReadFirstBySponsoringOrganizationId]') IS NOT NULL
-BEGIN
-    DROP PROCEDURE [dbo].[OrganizationSponsorship_ReadFirstBySponsoringOrganizationId];
-END
-GO
-
-CREATE PROCEDURE [dbo].[OrganizationSponsorship_ReadFirstBySponsoringOrganizationId]
-    @SponsoringOrganizationId UNIQUEIDENTIFIER
-AS
-BEGIN
-    SELECT TOP 1
-        *
-    FROM
-        [dbo].[OrganizationSponsorshipView]
-    WHERE
-        [SponsoringOrganizationId] = @SponsoringOrganizationId AND
-        [LastSyncDate] IS NOT NULL
-END
-GO
-
-
```

Comment:
I believe we still need this sproc, but it should be renamed `latest` instead of first and `ORDER BY LastSyncDate DESC`